lm_eval/tasks/spanish_bench/escola.yaml

```diff
@@ -0,0 +1,14 @@
+task: escola
+dataset_path: nbel/EsCoLA
+output_type: multiple_choice
+training_split: train
+validation_split: validation
```

Comment:
Looks like the dataset does not have a `validation` split.
https://huggingface.co/datasets/nbel/EsCoLA

Reply:
Thank you for bringing this to our attention. We've already contacted the dataset authors to fix this as soon as possible.

Reply:
Since the EsCoLA dataset authors cannot make the changes for a few weeks, we've decided to temporarily remove the task from the benchmark. We have made all the necessary updates. Thank you!